packages/oruga-next/src/components/loading/index.ts

```diff
@@ -3,40 +3,47 @@ import { createVNode, render } from "vue";
 
 import Loading from "./Loading.vue";
 
-import { VueInstance } from "../../utils/config";
-import { merge } from "../../utils/helpers";
+import { VueInstance } from "@/utils/config";
+import { merge } from "@/utils/helpers";
 import {
     registerComponent,
     registerComponentProgrammatic,
-} from "../../utils/plugins";
-import InstanceRegistry from "../../utils/InstanceRegistry";
+} from "@/utils/plugins";
+import InstanceRegistry from "@/utils/InstanceRegistry";
+
+declare module "@/types" {
+    interface OrugaPrgrammatic {
```

Comment:
There is this typo `OrugaPrgrammatic` in a few places

Reply:
@messenjer I appreciate the review. Typo is fixed :)